### PR TITLE
Solving questions

### PR DESCRIPTION
**Pull request by riibeirogabriel**
#### Changes
 

Fixes # (issue)

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

#### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by riibeirogabriel** [comment]
@LaChapeliere please take a look, i set the future work according with my monograph, but you can suggest modifications.

**Comment by LaChapeliere** [comment]
@riibeirogabriel Your doc says the hosts have to be Linux based but you wrote that the package can be used from any system, could you re-explain to me how it works please?

**Comment by riibeirogabriel** [comment]
The package can be runned on personal user machine, the requirement of linux machines is to cluster rented by user, where learningOrchestra will be deployed, the package use an ip from this cluster to makes the communication with microservices APIs. Do you understand now?

**Comment by LaChapeliere** [comment]
Let me rephrase to see if I got it. I need to deploy learningOrchestra on a Linux cluster, but then I can all its microservices from any computer as long as I have the python package/use the REST API  and an IP address for a cluster instance?

**Comment by riibeirogabriel** [comment]
> Let me rephrase to see if I got it. I need to deploy learningOrchestra on a Linux cluster, but then I can call its microservices from any computer as long as I have the python package and an IP adress for a cluster instance?

You can deploy the cluster with learningOrchestra on europe and use with python package or microservices APIs in America.

**Comment by LaChapeliere** [comment]
Tell me what you think of my changes

**Comment by riibeirogabriel** [comment]
Very good! please make the PR
